packages/pwa/app/utils/url.js

```diff
@@ -102,24 +102,40 @@ export const productUrlBuilder = (product) => encodeURI(`/product/${product.id}`
 export const searchUrlBuilder = (searchTerm) => `/search?q=${searchTerm}`
 
 /**
- * Returns a URL using the new locale.
+ * Returns a relative URL for a locale short code.
  *
- * @param previousLocale
- * @param newLocale
- * @returns {string}
+ * @param shortCode - The locale short code.
+ * @param opts - The options like `disallowParams` an array parameters to remove.
+ * @returns {string} - The relative URL for the specific locale.
  */
-export const buildUrlLocale = (previousLocale, newLocale) => {
+export const getUrlWithLocale = (shortCode, opts = {}) => {
```

Comment:
Can you extract changes related to `getUrlWithLocale` into a separate PR perhaps? I think they can stand on their own, and I'll need them too in my localization work: https://github.com/SalesforceCommerceCloud/pwa-kit/pull/137/files#diff-f46ab45bbea466e88ac5ab4c430662f8f265bce3754178d40040643bdfc72b3aR113

Reply:
The currency bug in the SCAPI `product-search` endpoint is fixed now, so we should be good to merge this PR.